app/views/custom/welcome/index.html.erb

```diff
@@ -20,6 +20,13 @@
   </div>
 
   <%# render Welcome::FinancedDomainsComponent.new %>
+
+  <div class="row homepage-main-content">
+    <div class="small-12 column ">
+      <%= raw content_block("homepage_main_content") %>
```

Comment:
To avoid the security concern raised by erb-lint:

```suggestion
      <%= AdminWYSIWYGSanitizer.new.sanitize(content_block("homepage_main_content")) %>
```

Which is consistent with the other places where we offer a CKEditor for administrators.